<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScalesDialog</class>
 <widget class="QDialog" name="ScalesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelButtons">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>605</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperament management:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>641</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizButtons">
    <item>
     <widget class="QPushButton" name="pushEditNotes">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Edit the selected temperament (in scale form)</string>
      </property>
      <property name="text">
       <string>&amp;Edit (as notes)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushEditFifths">
      <property name="toolTip">
       <string>Edit the selected temperament (in circle of fifths form)</string>
      </property>
      <property name="text">
       <string>Edit (as &amp;fifths)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushRestore">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Delete the selected temperament</string>
      </property>
      <property name="text">
       <string>Restore Default Scale</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushImport">
      <property name="toolTip">
       <string>Import a Scala scale file and add it to the collection</string>
      </property>
      <property name="text">
       <string>&amp;Import Scala file (*.scl)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>641</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizCommands">
    <item>
     <widget class="QCheckBox" name="checkUpdatePitches">
      <property name="text">
       <string>Update pitch if tuning is over 200</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushClose">
      <property name="toolTip">
       <string>Exit without applying any temperament</string>
      </property>
      <property name="text">
       <string>Cl&amp;ose</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="scaleName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushClose</sender>
   <signal>clicked()</signal>
   <receiver>ScalesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
